environment.yml

```diff
@@ -0,0 +1,21 @@
+name: autoqubo
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - dimod==0.10.17
+  - ipykernel
+  - black
```

Comment:
What is `black`, `pyjwt`, `pysciopt` and `auth0-python`  @lpodl ?

cc @gserban

Reply:
black is a formatter so we don't have any noise in our commits, so I suggest to leave that in there.

pyjwt and auth0 were necessary for the DA API if I remember correctly.
pyscipopt is for SCIP.
both of them should be removed, should I open another PR?

Reply:
yes please 🙏